src/snowflake/cli/plugins/connection/commands.py

```diff
@@ -210,9 +205,7 @@ def add(
     )
 
 
-@app.command()
-@global_options
-@with_output
+@app.command(requires_connection=False)
```

Comment:
Isn't `requires_connection` false by default?

Reply:
Yes, I just wanted to be explicit here